src/GraphQLMiddleware.php

```diff
@@ -67,12 +61,7 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
     }
 
     private function isGraphQLRequest(ServerRequestInterface $request) {
-        return $this->hasUri($request) || $this->hasGraphQLHeader($request);
```

Comment:
in other words, the middleware shoul always be activated and handle a request, but only if the uri of that request match with the one configured or the header is present. Hope i was clear.